src/operator/scan.ts

```diff
@@ -13,14 +13,15 @@ import {errorObject} from '../util/errorObject';
  * @returns {Obervable} An observable of the accumulated values.
  */
 export function scan<T, R>(accumulator: (acc: R, x: T) => R, seed?: T | R): Observable<R> {
-  return this.lift(new ScanOperator(accumulator, seed));
+  let _this: Observable<T> = this;
+  return _this.lift(new ScanOperator(accumulator, seed));
```

Comment:
That's true. The other side of that coin is that enforcing the `let _this: Observable<T>` pattern consistently will prevent people from making mistakes due to `this` being `any` down the line. I have no strong opinion either way but just wanted to point it out. /cc @kwonoj